src/ztc/api/serializers/zaken.py

```diff
@@ -54,6 +74,16 @@ def __init__(self, *args, **kwargs):
         self.fields["aard_relatie"].help_text += f"\n\n{value_display_mapping}"
 
 
+class CreateCustomUrlMethodField(SerializerMethodField):
+    def __init__(self, method_name=None, **kwargs):
```

Comment:
Ik zou dit mechanischme (iets) anders maken dan hoe hij nu is. Het veld zelf lijkt nu heel abstract maar de functie die je bijvoorbeeld opgeeft (`create_custom_urls`) kan alleen omgaan met een select aantal argumenten. Wat je hier in feite doet is het doorsturen van `**kwargs` voor het **veld** aan de opgegeven functie (via `method_name`). Met andere woorden; de functie die je opgeeft met `method_name` zou ook alle argumenten moeten ondersteunen van `SerializerMethodField`. Aan de functie argumenten te zien van `create_custom_urls` ondersteunt de functie dat niet.